test/unit/specs/SlideCard.spec.js

```diff
@@ -0,0 +1,59 @@
+import Vue from 'vue'
+import SlideCard from '@/components/SlideCard'
+
+describe('SlideCard.vue', () => {
+  let Constructor, vm
+
+  before(() => {
+    Constructor = Vue.extend(SlideCard)
+    vm = new Constructor().$mount()
+  })
+
+  it('has correct images', () => {
```

Comment:
In `Slides.spec` it appears that we are using a mock for testing, but for this spec file we are using hardcoded values from the file we're testing... We should be more consistent with how we are formatting our tests - I'm of the opinion mocks are nicer 😛

Reply:
None of the current PR's that are using Event/SlideCard, also I've changed the tests to use mocks